### PR TITLE
[actions] Extend type definitions

### PR DESCRIPTION
**Pull request by florian-h05**
This extends (or really enables) the type definitions for the `actions` namespace.

I needed to change the way of exporting the actions to have the type definitions generated.

Also, I added the NotificationAction using a try_catch statement to cover the (rare?) case that somebody does not have the Cloud Connector add-on installed. 

@digitaldan This is the PR I mentioned yesterday.

### CODE REVIEW

**Comment by florian-h05** [comment]
@digitaldan 
I am planning to publish a new version (2.0.1) soon after this PR gets merged to publish the two bugfixes I made as well as this enhancement.

File: actions.js [hunk 1]
```diff
@@ -253,10 +261,51 @@ const ThingsAction = Java.type('org.openhab.core.model.script.actions.Things');
...
+ * NotificationAction.sendLogNotification('<message>'); // only listed in the notification log
+ *
+ * @name NotificationAction
+ * @memberof actions
+ */
+let NotificationAction;
+try {
+  NotificationAction = Java.type('org.openhab.io.openhabcloud.NotificationAction');
+} catch (error) {
+  if (error.name !== 'TypeError') throw Error(error);
```
Comment (digitaldan) [inline actions.js:284]: I'm assuming if the cloud binding is not installed it will throw this 'TypeError' ?

**Review by digitaldan — APPROVED** [review]
LGTM!

File: actions.js (hunk 1 above)
Comment (florian-h05) [reply actions.js:284]: Yes.
When the binding is not installed, the loading of the class fails with the TypeError.

Comment (florian-h05) [reply actions.js:284]: This error is caught, other errors are rethrown.

**Pull request merged**